sonic-net/sonic-buildimage
# Support FIPS DB configuration

#### Why I did it
Support FIPS DB configuration
Design Doc: https://github.com/sonic-net/SONiC/pull/1372

##### Work item tracking
- Microsoft ADO **(number only)**: 24411148

#### How I did it
Add the FIPS Yang model to make FIPS configurable in ConfigDB.

#### How to verify it
See TestPlan: https://github.com/sonic-net/sonic-mgmt/pull/9092
Build the image and run the tests: https://github.com/sonic-net/sonic-mgmt/pull/9091



#### Which release branch to backport (provide reason below if selected)




#### Tested branch (Please provide the tested image version)




#### Description for the changelog




#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
